Sources/Valet/SinglePromptSecureEnclaveValet.swift

```diff
@@ -176,6 +198,9 @@ public final class SinglePromptSecureEnclaveValet: NSObject {
     @objc(allKeysWithUserPrompt:)
     public func allKeys(userPrompt: String) -> Set<String> {
```

Comment:
Might be worth noting the that an empty set will be returned when the keychain is not accessible.